.github/workflows/download-link-check-issue.yml

```diff
@@ -0,0 +1,94 @@
+name: Scheduled DVC Download Link Check
+
+on:
+  workflow_call:
+    inputs:
+      lychee_args:
+        description: The args to provide to lychee-action
+        required: true
+        type: string
+      owner:
+        description:
+          The owner of repository to operate on if the event doesn't include it
+        required: false
+        type: string
+        default: '${{ github.event.repository.owner.login }}'
+      repo:
+        description:
+          The repository to operate on if the event doesn't include it
+        required: false
+        type: string
+        default: '${{ github.event.repository.name }}'
```

Comment:
As far as I can tell from the docs, the `schedule` event doesn't include any `repository` info. I'll have to dump context on a schedule event to confirm that, but it's probably best to do it on an unrelated repo since I think the action has to be merged into main.

For now, manually passing in our repo info ensures this should work no matter the trigger event.